src/test/java/persistence/sql/dml/DeleteQueryBuilderTest.java

```diff
@@ -3,17 +3,30 @@
 import domain.Person;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
-import persistence.sql.ddl.QueryValidator;
-
-import java.util.List;
+import persistence.sql.metadata.EntityMetadata;
 
 import static org.junit.jupiter.api.Assertions.*;
 
 class DeleteQueryBuilderTest {
+	private final Person person = new Person(1L, "hhhhhwi", 1, "aab555586@gmail.com", 0);
+
+	private final EntityMetadata entityMetadata = new EntityMetadata(Person.class);
+
+	@DisplayName("Person 객체로 DELETE 쿼리 생성 테스트")
+	@Test
+	void test_buildQuery() {
+		assertEquals(
+				new DeleteQueryBuilder().buildQuery(entityMetadata),
+				"DELETE FROM users';"
```

Comment:
```suggestion
				"DELETE FROM users;"
```
오타가 있어서 테스트가 실패하네요 :)